release-process.md

```diff
@@ -75,9 +44,37 @@ For example if you are cutting RC for 1.0.2, mark such issues as `FIXED` in 1.0.
 release, and change them to the current release.
 - Verify from `git log` whether they are actually making it in the new RC or not.
 
-The process of cutting a release candidate has been automated via the AMPLab Jenkins. There are 
-Jenkins jobs that can tag a release candidate and create various packages based on that candidate. 
-The recommended process is to ask the previous release manager to walk you through the Jenkins jobs.
+The process of cutting a release candidate has been partially automated via the AMPLab Jenkins. There are
+Jenkins jobs that can tag a release candidate and create various packages based on that candidate.
```

Comment:
Even though they're not being used here and wont' mean anything to non-release-managers, is it worth mentioning what the jobs are named, just for future reference?